scripts/generate_carrier_encounter_events

```diff
@@ -5,22 +5,23 @@ THIS_SCRIPT_DIR="$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )"
 # source ${THIS_SCRIPT_DIR}/pipeline.sh
 ASSETS=${THIS_SCRIPT_DIR}/../assets
 ARGS=( \
-  SOURCE_TABLE \
+  SOURCE_QUERY \
   VESSEL_INFO \
   VESSEL_DATABASE \
   SPATIAL_MEASURES \
   COUNTRY_CODES \
   VOYAGES \
   NAMED_ANCHORAGES \
   FISHING_VESSELS \
+  MAX_MEDIAN_SPEED_KNOTS \
```

Comment:
@andres-arana I'm seeing this new parameter but I'm not seeing any modification in the dag to support it. Also, you need to include this in the CHANGES.md

Reply:
Yeah, check the PR now, I hadn't pushed the docs + airflow stuff before. Sorry about that.